net/uradvd/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_VERSION:=1
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Advertise an IPv6 prefix/route
+  DEPENDS:=+librt
+endef
+
+define Package/uradvd/description
+	Advertise an IPv6 prefix from the node
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	$(CP) ./src/* $(PKG_BUILD_DIR)/
```

Comment:
The source is pretty small and likely not to change too much, but is it available online ?
Maybe at least we should put a link to it.

Reply:
At the moment the primary source is in Gluon's OpenWrt feed at https://github.com/freifunk-gluon/packages/tree/master/net/uradvd. I see two options here:
- We drop the package from our feed, making openwrt/packages the primary source
- We create a separate repository for the source in the freifunk-gluon Github org

The latter is probably preferable. I can take care of creating the repo if we want to go with that.

Reply:
Sounds good to me. Please provide a link to get the source code from. :)

Reply:
I've pushed the code to https://github.com/freifunk-gluon/uradvd with minor Makefile improvements and a LICENSE file that can be referenced from the OpenWrt Makefile.

I don't think it makes sense to start versioning the source, so I propose we remove `PKG_VERSION` from the Makefile and use `PKG_SOURCE_DATE`-based package versions, as in https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/network/config/netifd/Makefile;h=c33e0c2134d6f53bd8bbfff66d58c605763e5fc8;hb=HEAD

Reply:
Please have a look. Do you want to be the maintainer? It would make sense. I do not care.

Reply:
With the removal of the source and most of the package Makefile, the only remaining part of the package is the initscript and the example config - both of which I don't care about much, so I'd prefer for you to keep the maintainership.

Reply:
fine with me, it is low maintenance.